.gitignore

```diff
@@ -25,3 +25,4 @@ tmp*.json
 *.xml
 *.zip
 *.tsbuildinfo
+/website/api
```

Comment:
What will this section be used for?

Reply:
Its where the generated typedoc API ends up